### ISSUE TITLE
Fails to activate

### ISSUE THREAD

**Issue by IzaacJ** [AdamRaichu/vscode-minecraft-lang#11]
Hello.
Installed this on a fresh install of VS Code (system-wide), and it fails to activate.

Getting this in the extension host log:

```
2022-11-18 11:51:49.697 [error] Activating extension adamraichu.minecraft-lang failed due to an error:
2022-11-18 11:51:49.697 [error] c:\Users\Izaac\.vscode\extensions\adamraichu.minecraft-lang-2.1.2\main.js:3
export function activate() {
^^^^^^

SyntaxError: Unexpected token 'export'
	at new Script (node:vm:100:7)
	at u._compile (c:\Program Files\Microsoft VS Code\resources\app\out\vs\loader.js:4:1173)
	at Module._extensions..js (node:internal/modules/cjs/loader:1220:10)
	at Module.load (node:internal/modules/cjs/loader:1035:32)
	at Module._load (node:internal/modules/cjs/loader:876:12)
	at c._load (node:electron/js2c/asar_bundle:5:13343)
	at a._load (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:118:14563)
	at v._load (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:113:64900)
	at w._load (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:113:64268)
	at Module.require (node:internal/modules/cjs/loader:1059:19)
	at require (node:internal/modules/cjs/helpers:102:18)
	at Function.r [as __$__nodeRequire] (c:\Program Files\Microsoft VS Code\resources\app\out\vs\loader.js:5:101)
	at h._loadCommonJSModule (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:113:66459)
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
	at async Promise.all (index 0)
```

VS Code: 1.73.1
Electron: 19.0.17
Node: 16.14.2

**Comment by AdamRaichu** [comment]
I am going to try removing the import and export statements from main.js. I do not have VS Code installed on my machine, so I am unable to test this myself, but I will release this in a pre-release (2.1.3). Could you install the pre-release and see if that fixes the problem?

**Comment by AdamRaichu** [comment]
@IzaacJ have you been able to try the pre-release version yet? If so and the problem was fixed, please close this issue.

**Comment by AdamRaichu** [comment]
I made a mistake and forgot to call `activate` after defining. The attempted fix for this will be 2.2.0.

**Comment by AdamRaichu** [comment]
Closing due to lack of response. Please comment if the issue persists.
